src/Shared/Controls/Storage/FolderBrowser.xaml.cs

```diff
@@ -94,8 +130,15 @@ private async Task OnCurrentFolderChanged(IFolder? oldValue, IFolder? newValue)
         if (newValue is null)
             return;
 
-        await foreach (var item in newValue.GetItemsAsync())
+        FetchingItems = true;
+        IsFolderEmpty = false;
+
+        await foreach (var item in newValue.GetItemsAsync(StorableType.All))
```

Comment:
Passing the default value isn't needed.

```suggestion
        await foreach (var item in newValue.GetItemsAsync())
```